event-subscription/service/example-event-activation.yaml

```diff
@@ -3,6 +3,6 @@ kind: EventActivation
 metadata:
   name: example-event-activation
   labels:
-    example: event-bus    
+    example: event-bus-service-subscription
 spec:
   sourceId: stage.commerce.kyma.local
```

Comment:
Should we consider not using `commerce`?

Reply:
I renamed it to "external-application"